tests/dummy/app/templates/ember-flexberry-dummy-comment-edit.hbs

```diff
@@ -1,4 +1,10 @@
 <h3 class="ui header">{{t "forms.ember-flexberry-dummy-comment-edit.caption"}}</h3>
+<div class="field">
```

Comment:
In Semantic UI fields are available only inside forms: http://semantic-ui.com/collections/form.html#field